config/application.rb

```diff
@@ -45,7 +45,11 @@
 require_relative '../lib/active_record/batches'
 require_relative '../lib/simple_navigation/item_extensions'
 
-Dotenv::Railtie.load
```

Comment:
This tells dotenv to overwrite env vars.  Because vagrant defines LOCAL_DOMAIN in the .env.vagrant file, I need this to make tests pass locally.